plugins/poll/spec/lib/votes_updater_spec.rb

```diff
@@ -1,94 +0,0 @@
-require 'rails_helper'
-
-describe DiscoursePoll::VotesUpdater do
```

Comment:
Ditto https://github.com/discourse/discourse/pull/6359/files#r215156653

Reply:
We don't update the votes anymore. We delete them whenever a poll is changed.